src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Renamed.cs

```diff
@@ -101,4 +101,130 @@ public static void FileSystemWatcher_Renamed_FileInNestedDirectory()
             }
         });
     }
+
+    [Fact]
+    [ActiveIssue(2545)]
+    [PlatformSpecific(PlatformID.Windows | PlatformID.OSX)]
```

Comment:
This was originally not linux since you can't do a Directory.Move on default Ubuntu since the system Temp dir is on a different mount point; I'll switch this to not use the system Temp and use a test-specific temp so this can be enabled on Linux